packages/modules/src/firebase/serve-modules-api.ts

```diff
@@ -18,7 +18,10 @@ export default (req: Request, res: Response) => {
   if (url.endsWith('.json')) {
     url = url.slice(0, -5);
   }
+  // eslint-disable-next-line prefer-destructuring
+  url = url.split('?')[0];
```

Comment:
```suggestion
  [url] = url.split('?');
```

Em vez de desabilitar a rule 🙃